src/components/augmint-ui/FormCustomLabel/styles.js

```diff
@@ -9,5 +9,5 @@ const BaseLabel = `
 
 export const StyleLabel = styled.label`
     ${BaseLabel};
-    border-radius: ${props => (props.align === "right" ? "0 5px 5px 0" : "")};
+    border-radius: ${props => (props.align === "right" ? "0 5px 5px 0" : props.align === "left" ? "5px 0 0 5px" : "")};
```

Comment:
this border radius appears in multiple places, it would make sense to move it to the theme

Reply:
done